contents/blog/the-posthog-array-1-41-0.md

```diff
@@ -0,0 +1,168 @@
+---
+date: 2022-11-02
+title: "Array 1.41.0: Improving performance by up to 400%"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ['Release notes', 'Product updates']
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+---
+
+Want to know more about what we're up to? [Subscribe to HogMail, our newsletter](/newsletter), which we send every two weeks!
+
+## 1.41.0 update guide for self-hosted users
+Due to changes in this update, it is important check `alias` usage before [upgrading PostHog](/docs/runbook/upgrading-posthog) to 1.41.0 on a self-hosted instance. For example, assuming `email` is used as the identified user id, then: 
```

Comment:
might be nice to link to https://posthog.com/docs/integrate/identifying-users#considerations e.g as "read more"

Reply:
```suggestion
Due to changes in this update, it is important check `alias` usage before [upgrading PostHog](/docs/runbook/upgrading-posthog) to 1.41.0 on a self-hosted instance. Further [information is available in the docs](/docs/integrate/identifying-users#considerations), but for example, assuming `email` is used as the identified user id, then: 
```